skimage/morphology/rolling_ball.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+
+import numpy as np
+
+from skimage.util import invert
+
+
+def rolling_ball(input_img, radius=50, white_background=False):
+    """Perform background subtraction using the rolling ball method.
+
+    The rolling ball filter is a segmentation method that aims to separate the
+    background from a grayscale image in case of uneven exposure. It is
+    frequently used in biomedical image processing and was first proposed by
+    Stanley R. Sternberg (1983) in the paper Biomedical Image Processing [1]_.
+
+    Parameters
+    ----------
+    image : ndarray
+        The image to be filtered.
+    radius : float
+        The radius of the ball that is used as the structuring element.
+    white_background : bool
+        If true, the algorithm separates dark features from a bright
+        background.
+
+    Returns
+    -------
+    filtered_image : ndarray
+        The image with background removes
+    background : ndarray
+        The background that was removed.
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from skimage import data
+    >>> from skimage.morphology import rolling_ball
+
+    Subtract a black background from an example image
+
+    >>> image = data.coins()
+    >>> result, bg = rolling_ball(image, radius=100)
+
+    Subtract a white background from an example image
+
+    >>> image = data.page()
+    >>> result, bg = rolling_ball(image, radius=100, white_background=True)
+
+    Notes
+    -----
+    Check out the gallery example of this function (TODO: link)
```

Comment:
This is not necessary. Links are automatically generated :)

Reply:
fixed